src/main/java/edu/cornell/kfs/module/ar/service/impl/CuContractsGrantsInvoiceCreateDocumentServiceImpl.java

```diff
@@ -245,12 +489,37 @@ protected void generateAndSaveContractsAndGrantsInvoiceDocument(ContractsAndGran
         }
     }
     
+    private Collection<ContractsAndGrantsBillingAward> setContractGrantInvoiceDocumentCreationProcessTypeCodeOnEachAward(Collection<ContractsAndGrantsBillingAward> awards, String creationProcessTypeCode) {
```

Comment:
Should this function return void?  Since the function is updating values in the collection of .awards passed in, the caller already has a reference to the collection.